setup.md

```diff
@@ -0,0 +1,92 @@
+### Install mbed-cli to a venv
+
+Install mbed-cli to a virtual environment, so that all the things mbed-cli
+installs automatically don't interfere with other things you have installed as
+part of your main environment (which may be managed by your package manager,
+etc.).
+
+```sh
+virtualenv venvs/mbed
+source ~/venvs/mbed/bin/activate
+pip install mbed-cli
+```
+
+### Install the GCC_ARM toolchain
+
+We'll install arm-none-eabi-gcc to `/opt`. You may want to use `~/.local` or
+some other place. Where doesn't matter too much, so long as its in your `$PATH`.
+
+```sh
+wget -q https://developer.arm.com/-/media/Files/downloads/gnu-rm/5_4-2016q3/gcc-arm-none-eabi-5_4-2016q3-20160926-linux.tar.bz2 -O gcc-arm-none-eabi-5_4-2016q3-20160926-linux.tar.bz2 && tar -xjf gcc-arm-none-eabi-5_4-2016q3-20160926-linux.tar.bz2 -C /opt
+rm gcc-arm-none-eabi-5_4-2016q3-20160926-linux.tar.bz2
+```
+
+Remember to add `/opt/gcc-arm-none-eabi-5.4-2016q3/bin` to your `$PATH`.
+
+### Install the ATECC608A example
+
+```sh
+git clone git@github.com:ARMmbed/mbed-os-example-atecc608a.git
+cd mbed-os-example-atecc608a/
+source ~/venvs/mbed/bin/activate
+mbed new .
+mbed deploy
+mbed compile -t GCC_ARM -m K64F --flash --sterm
+```
+
+### PSA Crypto driver development
+
+Add any driver files (none created so far) to the mbed-os-atecc608a folder. It
+is backed by the
+[mbed-os-atecc608a](https://github.com/ARMmbed/mbed-os-atecc608a/tree/mbed-cryptoauthlib)
```

Comment:
This is a link to private repository in a public file. Most people can't follow this link.

Reply:
I've made the repo public.